substrate/primitives/keystore/src/testing.rs

```diff
@@ -298,6 +310,15 @@ impl Keystore for MemoryKeystore {
 		self.sign::<bls381::Pair>(key_type, public, msg)
 	}
 
+	#[cfg(feature = "bls-experimental")]
+	fn bls381_generate_pop(
+		&self,
+		key_type: KeyTypeId,
+		public: &bls381::Public,
```

Comment:
`sp-io` crate doesn't build with `bls-experimental` feature enabled.
Maybe you need to wrap some of the params as the other methods?